tests/unit-tests/jobs/sequelize/sequelizeConnector-test.js

```diff
@@ -102,7 +102,9 @@ describe('Sequelize client tests', function () {
                 webhooks: ['http://zooz.com', 'http://payu.com'],
                 parallelism: 4,
                 max_virtual_users: 100,
-                notes: 'some nice notes'
+                notes: 'some nice notes',
+                proxy_url: 'http://proxy.com',
```

Comment:
create an integration test for `jobs` with proxy url and debug to make sure that the migrations worked and that these fields exist in the db

Reply:
done